src/FSharp.Data.GraphQL.Shared/TypeSystem.fs

```diff
@@ -1007,7 +1011,9 @@ and [<CustomEquality; NoComparison>] ScalarDefinition<'Val> =
       CoerceInput : Value -> 'Val option
       /// A function used to set a surrogate representation to be
       /// returned as a query result.
-      CoerceValue : obj -> 'Val option }
+      CoerceValue : obj -> 'Val option
+
+      Serialize : 'Val -> Value }
```

Comment:
@njlr what XML comment should we put here?

Reply:
Perhaps this?

> `Serialize` converts the .NET CLR type into a GraphQL AST value